tests/e2e/specs/frontend/legacy-template-blocks.test.js

```diff
@@ -43,7 +43,7 @@ describe( 'Legacy Template blocks', () => {
 	} );
 
 	afterAll( async () => {
-		await activateTheme( 'twentytwentyone' );
+		await activateTheme( 'storefront' );
```

Comment:
Switching theme to twentytwentyone makes the following tests fail, as most tests expect storefront as the theme. I will update this in a future PR to use the `useTheme` hook as @tomasztunik pointed out